elastic/kibana
# [Guided onboarding] Add guided onboarding to the home page

## Summary

Fixes https://github.com/elastic/kibana/issues/138654
Fixes https://github.com/elastic/kibana/issues/138550

This PR adds guided onboarding changes to the home page for Cloud deployments:
- The welcome screen is replaced with the guided onboarding landing page
- A guided onboarding link is added to the integrations section

### How to test
1. Add `xpack.cloud.id: 'testID'` to your `kibana.dev.yml` file to imitate a Cloud deployment
2. Remove `home:welcome:show` value from your `localStorage`
3. Open Kibana and check that you're correctly redirected to the guided onboarding landing page
4. Navigate to the home page and check the new guided onboarding link

### Screenshots

#### Welcome screen for self managed 

<img width="652" alt="Screenshot 2022-10-25 at 12 26 18" src="https://user-images.githubusercontent.com/6585477/197750988-a2cfcc7a-b47b-4ef5-aa49-6ae88916b481.png">


#### Welcome screen for Cloud deployments (guided onboarding landing page)

<img width="1245" alt="Screenshot 2022-10-24 at 15 02 46" src="https://user-images.githubusercontent.com/6585477/197531782-79776abb-200c-4acb-9bd5-bf35a78f2d92.png">


#### Home page with guided onboarding link (large screens)

<img width="1634" alt="Screenshot 2022-10-25 at 12 24 10" src="https://user-images.githubusercontent.com/6585477/197751101-6c103deb-14db-4b71-801e-e8bc099575e1.png">



#### Home page with guided onboarding link (small screens)

<img width="666" alt="Screenshot 2022-10-25 at 12 24 33" src="https://user-images.githubusercontent.com/6585477/197751285-edbe30c7-a97d-47a5-9618-1123f6673d77.png">




### Checklist
